dtm-database-backend/src/main.rs

```diff
@@ -1,3 +1,48 @@
-fn main() {
-  println!("Hello, world!");
+use axum::{routing::get, Router};
+use std::net::{IpAddr, Ipv4Addr, SocketAddr};
+
+const DEFAULT_PORT: u16 = 3001;
+const PORT_ENV: &str = "PORT";
+
+#[tokio::main]
+async fn main() {
+  // set port, ipv4, and socket.
+  let port: u16 = set_port(PORT_ENV);
+  let ipv4: Ipv4Addr = Ipv4Addr::UNSPECIFIED;
+  let socket_v4: SocketAddr = SocketAddr::new(IpAddr::V4(ipv4), port);
```

Comment:
```suggestion
  let socket_v4: SocketAddr = SocketAddr::from((Ipv4Addr::UNSPECIFIED, set_port(PORT_ENV)));
```
好みの域だけど、こんなふうに簡潔にできるよ〜〜っていうご提案